resources/windows/wix/customactions.wxs

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Include_CustomActions" />
+
+    <CustomAction Id="SaveCmdLineInstallDir"
+      Property="CMDLINE_INSTALLDIR"
+      Value="[INSTALLDIR]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetFromCmdLineInstallDir"
+      Property="INSTALLDIR"
+      Value="[CMDLINE_INSTALLDIR]"
+      Execute="firstSequence" />
+
+    <!-- PUPPET_MASTER_SERVER -->
+    <CustomAction Id="SaveCmdLinePuppetMasterServer"
+      Property="CMDLINE_PUPPET_MASTER_SERVER"
+      Value="[PUPPET_MASTER_SERVER]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetFromCmdLinePuppetMasterServer"
+      Property="PUPPET_MASTER_SERVER"
+      Value="[CMDLINE_PUPPET_MASTER_SERVER]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetDefaultPuppetMasterServer"
+      Property="PUPPET_MASTER_SERVER"
+      Value="puppet"
+      Execute="firstSequence" />
+
+    <!-- PUPPET_AGENT_ENVIRONMENT -->
+    <CustomAction Id="SaveCmdLinePuppetAgentEnvironment"
+      Property="CMDLINE_PUPPET_AGENT_ENVIRONMENT"
+      Value="[PUPPET_AGENT_ENVIRONMENT]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetFromCmdLinePuppetAgentEnvironment"
+      Property="PUPPET_AGENT_ENVIRONMENT"
+      Value="[CMDLINE_PUPPET_AGENT_ENVIRONMENT]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetDefaultPuppetAgentEnvironment"
+      Property="PUPPET_AGENT_ENVIRONMENT"
+      Value="production"
+      Execute="firstSequence" />
+
+    <!-- PUPPET_AGENT_CERTNAME -->
+    <CustomAction Id="SaveCmdLinePuppetAgentCertname"
+      Property="CMDLINE_PUPPET_AGENT_CERTNAME"
+      Value="[PUPPET_AGENT_CERTNAME]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetFromCmdLinePuppetAgentCertname"
+      Property="PUPPET_AGENT_CERTNAME"
+      Value="[CMDLINE_PUPPET_AGENT_CERTNAME]"
+      Execute="firstSequence" />
+
+    <!-- PUPPET_CA_SERVER -->
+    <CustomAction Id="SaveCmdLinePuppetCaServer"
+      Property="CMDLINE_PUPPET_CA_SERVER"
+      Value="[PUPPET_CA_SERVER]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetFromCmdLinePuppetCaServer"
+      Property="PUPPET_CA_SERVER"
+      Value="[CMDLINE_PUPPET_CA_SERVER]"
+      Execute="firstSequence" />
+
+    <!-- PUPPET_AGENT_STARTUP_MODE -->
+    <CustomAction Id="SaveCmdLinePuppetAgentStartupMode"
+      Property="CMDLINE_PUPPET_AGENT_STARTUP_MODE"
+      Value="[PUPPET_AGENT_STARTUP_MODE]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetFromCmdLinePuppetAgentStartupMode"
+      Property="PUPPET_AGENT_STARTUP_MODE"
+      Value="[CMDLINE_PUPPET_AGENT_STARTUP_MODE]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetDomainToLocalComputer"
```

Comment:
I think `<!-- PUPPET_AGENT_ACCOUNT_DOMAIN -->` above this line got lost somewhere along the line. I'd like to add it back since these comments break up the content of this file in a way that makes it easier to parse